lib/miq_expression.rb

```diff
@@ -678,7 +685,7 @@ def self.quote_human(val, typ)
     when "string", "date", "datetime"
       return "\"#{val}\""
     else
-      return quote(val, typ)
+      quote(val, typ)
```

Comment:
If you are killing the return here, you can kill it in every other conditional above